modules/atomic-opt-in/assets/js/welcome-screen/welcome-popover.js

```diff
@@ -40,16 +40,13 @@ export const WelcomePopover = ( { doClose } ) => {
 	}
 
 	return (
-		<Popover
+		<Dialog
 			open={ Boolean( anchorElRef.current ) }
 			onClose={ doClose }
-			anchorEl={ anchorElRef.current }
-			anchorOrigin={ { vertical: 'center', horizontal: 'center' } }
-			transformOrigin={ { vertical: 'center', horizontal: 'center' } }
-			slotProps={ {
-				paper: {
-					sx: { width: 600 },
-				} } }
+			maxWidth="sm"
+			PaperProps={ {
+				sx: { borderRadius: '8px' },
+			} }
```

Comment:
Already handled in the new theme, can be removed.
+
Please change the Button color to "accent" to make it pink (instead of the default black primary).